frontend/src/components/core/Block/Block.tsx

```diff
@@ -237,14 +258,31 @@ class Block extends PureComponent<Props> {
         <ExceptionElement element={el} width={width} />
       ),
       graphvizChart: (el: SimpleElement) => (
-        <GraphVizChart element={el} index={index} width={width} />
+        <FullScreenWrapper width={width}>
+          {({ width, height }) => (
+            <GraphVizChart
+              element={el}
+              index={index}
+              width={width}
+              height={height}
+            />
+          )}
+        </FullScreenWrapper>
+      ),
+      imgs: (el: SimpleElement) => (
+        <FullScreenWrapper width={width}>
+          {({ width, height }) => <ImageList element={el} width={width} />}
```

Comment:
No height for images?

Reply:
Is not necessary, each image handle her own height (to do not lose aspect relation)